### DISCUSSION TITLE
why no -d ?

### DISCUSSION THREAD

**Discussion by simonmcnair** [AbdBarho/stable-diffusion-webui-docker#574] [General]
why does the documentation not recommend ```docker compose --profile [ui] up --build -d``` or at least explain or recommend post verification that it works ?

Otherwise a disconnect from the shell and your instance is gone ?

Just curious

**Comment by AbdBarho** [comment]
If this is your first time using docker and this repo, you won't see any errors / logs with `-d` and you would not know what to do next.

Any additional documentation in this repo is redundant because it is general docker / shell stuff, and there are many resources online that do better job than me trying to explain.
